### DISCUSSION TITLE
update database nominatim

### DISCUSSION THREAD

**Discussion by martimlello07** [osm-search/Nominatim#2972] [General]
Hi everyone,
I am adding new databases with nominatim add -data. I first added Portugal and now I am adding Spain. Through this method ("add -data") will I be limited later if I want to implement a daily update or can I still do it? I ask this because on the API site, there is a topic of adding multiple regions with updates and im not using that option.

Thank you

**Comment by mtmail** [comment]
Good to see you saw the topic of adding multiple regions in the documentation. That's still the case and still complicated. We don't have an easier solution yet.

**Reply by martimlello07** [reply]
yes. but can I implement the database update later when using this method?

**Reply by mtmail** [reply]
That should still work. Keep a copy of the Portugal data file (pbf) that you initially imported, or any other country you import.

Later when you want to add updates create the directory structure https://nominatim.org/release-docs/latest/admin/Advanced-Installations/#importing-multiple-regions-with-updates

Then you should run only parts of `import_multiple_regions.sh` script: you don't want to actually import anything, so remove the
```
echo "Setting up nominatim db"
${IMPORT_CMD} 2>&1
```
lines.

Overwrite the *.pbf files `wget` downloaded with your own files. Run `pyosmium-get-changes` again to find the sequence number and overwrite the `sequence.state` files, too.

**Reply by martimlello07** [reply]
hi.
when i execute the script update_database.sh , it gives en error saying "The requested URL was not found on this server."
when i go to the directory website , the file update.php is not there.

can u help me with this problem?

**Reply by mtmail** [reply]
>  when i go to the directory website , the file update.php is not there.

The documentation is clear that the files are in the project directory, not website directory. The documentation does not mention a 'update.php'. You execute the script on the command-line, not via a web browser.

**Reply by martimlello07** [reply]
im executing the script on the command-line but it is giving that error ("The requested URL was not found on this server")

**Reply by martimlello07** [reply]
how could i solve this problem?

**Reply by mtmail** [reply]
I don't know the URL, the server or your configuration. You will need to read the script, find the line that is printing the output. Then find the URL that is missing.

**Reply by mtmail** [reply]
You changed the FOLLOWUP variable. See the original script https://github.com/osm-search/Nominatim/blob/master/utils/update_database.sh
